shared-bindings/usb_cdc/Serial.c

```diff
@@ -64,6 +63,28 @@
 //|         :rtype: bytes"""
 //|         ...
 //|
+//|     def readline(self, size=-1) -> Optional[bytes]:
+//|         r"""Read a line, ending in a newline character ("\\n"), or
+//|         return everything readable if no newline is found and ``timeout`` is 0.
+//|         Return ``None`` in case of error.
+//|
+//|         This is a binary stream: the newline character cannot be changed.
+//|
+//|         :param int size: maximum number of characters to read. ``-1`` means as many as possible.
+//|         :return: the line read
```

Comment:
The read line will include the `\r` character at the end:
![image](https://user-images.githubusercontent.com/10207857/110255359-27368c00-7f8b-11eb-9a16-a94079c06d9b.png)

Reply:
Yes, the newline character is always `b'\n'` and `b'\n'` and it is preserved (if present) in the return value of readline.  It might be missing if the `readline()` call terminates due to timeout instead, so it is also useful. From what I can see, this matches pyserial.

Reply:
I know, just pointing this out as it isn't described in the file.

Reply:
I will add one more sentence making this super-clear.